DevCycle.SDK.Server.Local/Api/LocalBucketing.cs

```diff
@@ -51,7 +61,7 @@ public LocalBucketing()
                 "env",
                 "abort",
                 Function.FromCallback(WASMStore,
-                    (Caller caller, int messagePtr, int filenamePtr, int linenum, int colnum) =>
+                    (Caller caller, int messagePtr, int filenamePtr, int linenum, int column) =>
```

Comment:
this wasn't a typo, it's the column number; but I guess this works too.

Reply:
oh lol